### ISSUE TITLE
How to get the COCO style json file for MHP or CHIP dataset?

### ISSUE THREAD

**Issue by zyf12389** [soeaver/Parsing-R-CNN#5]
annotations style in COCO must be binary. So, what is your json file generated by, is it based on the whole person or the body part? Thanks.

**Comment by jiawenhao2015** [comment]
the same question~~~~thanks

**Comment by jiawenhao2015** [comment]
> annotations style in COCO must be binary. So, what is your json file generated by, is it based on the whole person or the body part? Thanks.

hello~~~do you find the way get the json anno file after 1 year?......many thanks.~~~~~

**Comment by zyf12389** [comment]
I just added the path of the annotation to the json file in coco format.(e.g.
annotation_info = {
        "id": annotation_id,
        "image_id": image_id,
        "bbox": bounding_box.tolist(),
        "parsing_file": annotation_filename,
        ......
    })

**Comment by jiawenhao2015** [comment]
> I just added the path of the annotation to the json file in coco format.(e.g.
> annotation_info = {
> "id": annotation_id,
> "image_id": image_id,
> "bbox": bounding_box.tolist(),
> "parsing_file": annotation_filename,
> ......
> })



many many thanks for your reply~~~~
so you mean, the json file actually contains no label info, but the really label img's path?
do you change the dataset.py code? or use some other tools convert the MHP or CHIP label to coco style?
~~~~~~~thanks you again~~~~~

**Comment by zyf12389** [comment]
Yes, the json file actually contains label annotation's path and used[ pycococreator](https://github.com/waspinator/pycococreator).

**Comment by jiawenhao2015** [comment]
> Yes, the json file actually contains label annotation's path and used[ pycococreator](https://github.com/waspinator/pycococreator).

ok ~~~~~~thanks a lot~~~~~~~~~~~i will try ~~~
